modules/arm_plugin/armplg_build.sh

```diff
@@ -74,11 +74,11 @@ checkSrcTree()
 
 
 #Prepare sources
-checkSrcTree $OPENCV_HOME https://github.com/opencv/opencv.git 4.5.3-openvino master
-checkSrcTree $OPENVINO_HOME https://github.com/openvinotoolkit/openvino.git 2021.4 releases/2021/4
-checkSrcTree $OPENVINO_CONTRIB https://github.com/openvinotoolkit/openvino_contrib.git 2021.4 releases/2021/4
+checkSrcTree $OPENCV_HOME https://github.com/opencv/opencv.git 4.5.3-openvino-2021.4.1 master
+checkSrcTree $OPENVINO_HOME https://github.com/openvinotoolkit/openvino.git 2021.4.1 releases/2021/4
+checkSrcTree $OPENVINO_CONTRIB https://github.com/openvinotoolkit/openvino_contrib.git 2021.4.1 releases/2021/4.1
```

Comment:
There is no 2021.4.1 tag yet and branch releases/2021/4.1 looks outdated. Also it looks a bit inconsistent if contrib use 4.1 branch while main repository stick to branch 4.